ui/public/index.html

```diff
@@ -16,6 +16,7 @@
 		rel="stylesheet">
 	<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.9.3/Chart.min.js" integrity="sha512-s+xg36jbIujB2S2VKfpGmlC3T5V2TF3lY48DX7u2r9XzGzgPsa6wTpOQA7J9iffvdeBN0q9tKzRxVxw1JviZPg==" crossorigin="anonymous"></script>
 	<script defer src="/build/bundle.js"></script>
+	<script src="https://kit.fontawesome.com/20a9e23905.js" crossorigin="anonymous"></script>
```

Comment:
why do we need fontawesome?

Reply:
This is for the icons used in the some of the pages